### PR TITLE
Add plotting for sample logs

### PR DESCRIPTION
**Pull request by AndreiSavici**
Add possibility to plot sample logs.

**To test:**
Check documentation, see if you can run according to the instructions
Using files from DocTests and SystemTests
```
import matplotlib.pyplot as plt
from mantid import plots

w=Load('CNCS_7860')
fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w,LogName='ChopperStatus5')
fig.show()

fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w,LogName='ChopperStatus5', StartFromLog=True)
fig.show()

fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w,LogName='ChopperStatus5', FullTime=True)
fig.show()

w1=Load('HYS_13658')
fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w1,LogName='s1',StartFromLog=True,FullTime=True)
fig.show()
fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w1,LogName='s1')
fig.show()

fig = plt.figure()
ax1 = fig.add_subplot(211,projection='mantid')
ax2 = fig.add_subplot(212,projection='mantid')
ax1.plot(w, LogName='ChopperStatus5')
ax1.set_title('From run start')
ax2.plot(w, LogName='ChopperStatus5',FullTime=True)
ax2.title('Absolute time')
fig.tight_layout()
fig.show()
```

No issue or release notes

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/PythonInterface/mantid/plots/helperfunctions.py [hunk 1]
```diff
@@ -20,8 +20,7 @@
 import datetime
 from mantid.dataobjects import EventWorkspace, Workspace2D, MDHistoWorkspace
 from mantid.kernel import FloatTimeSeriesProperty
-import mantid.api.MDNormalization
-import mantid.kernel.config
+import mantid
```
Comment (peterfpeterson) [inline Framework/PythonInterface/mantid/plots/helperfunctions.py]: Change this to `import mantid.api, mantid.kernel`

**Review by peterfpeterson — CHANGES_REQUESTED** [review]
Looks close. It would help to have an extra example for having the two axes (absolute and relative) on the same plot.

**Comment by AndreiSavici** [comment]
Here is the extra example
```
import matplotlib.pyplot as plt
from mantid import plots

w=Load('CNCS_7860')
fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
ax.plot(w,LogName='ChopperStatus5')
axt=ax.twiny()
plots.plotfunctions.plot(axt,w,LogName='ChopperStatus5', FullTime=True)
fig.show()
```

Note that the twin axes do not accept the projection keyword

**Review by peterfpeterson — APPROVED** [review]
Looks good. I wonder if you should expand the check to include `Int32TimeSeriesProperty` and `Int64TimeSeriesProperty`. I'll assume that the `twiny` example will go in the plot gallery.

**Review by martyngigg — COMMENTED** [review]
Should we have some tests for this?

**Pull request merged**